### PR TITLE
Spruce up history and FODC table formatting

### PR DESCRIPTION
**Pull request by kitsuta**
Fixes https://github.com/magfest/ubersystem/issues/2139. The tables are now much more readable.

### CODE REVIEW

File: uber/templates/groups/history.html [hunk 1]
```diff
@@ -30,14 +30,36 @@
 {% endfor %}
 </table>
 
+    {% if group.stripe_transactions %}
```
Comment (kitsuta) [inline uber/templates/groups/history.html]: Whoops, this is from https://github.com/magfest/ubersystem/pull/2471. There will probably be a merge conflict when we merge that PR, I'll handle it when it comes up.

Comment (RobRuana) [inline uber/templates/groups/history.html]: I'd delete this change in another commit on this branch. If this somehow gets merged and #2471 doesn't get merged, then `group.stripe_transactions` will blow up, right?

Comment (kitsuta) [inline uber/templates/groups/history.html]: I am... not sure. In any case, yeah, if #2471 gets held up this should be excised. I was thinking we'd just wait to merge this until after that PR is merged.

Comment (RobRuana) [inline uber/templates/groups/history.html]: Sure. As long as this conversation is here to remind us of the dependency it should be fine.

**Comment by kitsuta** [comment]
We've merged https://github.com/magfest/ubersystem/pull/2471 so this is good to go now.

**Pull request merged**